### PR TITLE
doc/radosgw: edit "Usage" admin.rst

### PR DESCRIPTION
**Pull request by zdover23**
Edit "Usage" in doc/radosgw/admin.rst.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: doc/radosgw/admin.rst [hunk 1]
```diff
@@ -827,53 +827,63 @@ all unauthenticated users:
 Usage
 =====
 
-The Ceph Object Gateway logs usage for each user. You can track
-user usage within date ranges too.
+The Ceph Object Gateway logs the usage of each user. You can track the usage of
+each user within date ranges.
```
Comment (anthonyeleven) [inline doc/radosgw/admin.rst:831]: Suggest singular

File: doc/radosgw/admin.rst [hunk 2]
```diff
@@ -827,53 +827,63 @@ all unauthenticated users:
...
 
-- Add ``rgw_enable_usage_log = true`` in [client.rgw] section of ceph.conf and restart the radosgw service. 
+- Add ``rgw_enable_usage_log = true`` in the ``[client.rgw]`` section of
```
Comment (anthonyeleven) [inline doc/radosgw/admin.rst:833]: Until we get a linkable macro to handle the various ways of setting options, suggest we just say to set this true in central config or Ceph.conf and restart all RGWs

File: doc/radosgw/admin.rst [hunk 3]
```diff
@@ -827,53 +827,63 @@ all unauthenticated users:
...
   (**format:** ``yyyy-mm-dd [HH:MM:SS]``). 
   
 - **Log Entries:** The ``--show-log-entries`` option allows you to specify
-  whether or not to include log entries with the usage stats 
+  whether to include log entries with the usage stats 
   (options: ``true`` | ``false``).
 
-.. note:: You may specify time with minutes and seconds, but it is stored 
-   with 1 hour resolution.
+.. note:: You can specify time to a granularity of minutes and seconds, but the
```
Comment (anthonyeleven) [inline doc/radosgw/admin.rst:850]: Precision not granularity.

**Review by anthonyeleven — APPROVED** [review]

File: doc/radosgw/admin.rst (hunk 1 above)
Comment (zdover23) [reply doc/radosgw/admin.rst:831]: Accepted.

File: doc/radosgw/admin.rst (hunk 2 above)
Comment (zdover23) [reply doc/radosgw/admin.rst:833]: Accepted.

File: doc/radosgw/admin.rst (hunk 3 above)
Comment (zdover23) [reply doc/radosgw/admin.rst:850]: Accepted.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/55321 - Reef backport
https://github.com/ceph/ceph/pull/55322 - Quincy backport

**Pull request merged**